### PR TITLE
{tools}[system/system] vsc-mympirun v5.0.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2439.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e2778f53f9b87a4ceea896978c534ac6 for a full test report.

File: easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-5.0.1.eb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    }),
+    ('vsc-install', '0.15.0', {
+        'modulename': 'vsc.install',
+        'checksums': ['d6709feaf5451ec005381e5787a0086a35323efa4f6f0ead2b408cdffac15144'],
+    }),
+    ('vsc-base', '2.9.6', {
+        'modulename': 'vsc.utils',
+        'checksums': ['a62158022c36b5b456771ee8145d9ea3d299b301240e9724d04052e6aa0400b3'],
+    }),
+    (name, version, {
```
Comment (smoors) [inline easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-5.0.1.eb]: @boegel now it's you who removed the comment from the previous version :)
or does it not apply anymore?

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node377.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/489044f21e0f184aebedc61329fb5ccf for a full test report.

**Comment by boegel** [comment]
superseded by #10887...

**Pull request closed without merging**